content/en/docs/installing-on-linux/on-premises/install-kubesphere-on-vmware-vsphere.md

```diff
@@ -0,0 +1,401 @@
+---
+title: "VMware vSphere Installation"
+keywords: 'kubernetes, kubesphere, VMware vSphere, installation'
+description: 'How to install KubeSphere on VMware vSphere Linux machines'
+
+
+weight: 2260
+---
+
+# Introduction
+
+For a production environment, we need to consider the high availability of the cluster. If the key components (e.g. kube-apiserver, kube-scheduler, and kube-controller-manager) are all running on the same master node, Kubernetes and KubeSphere will be unavailable once the master node goes down. Therefore, we need to set up a high-availability cluster by provisioning load balancers with multiple master nodes. You can use any cloud load balancer, or any hardware load balancer (e.g. F5). In addition, Keepalived and [HAproxy](https://www.haproxy.com/), or Nginx is also an alternative for creating high-availability clusters.
+
+This tutorial walks you through an example of how to create [keepalived + haproxy](https://kubesphere.com.cn/forum/d/1566-kubernetes-keepalived-haproxy), and implement high availability of master and etcd nodes using the load balancers.
+
+## 1. Prerequisites
+
+- Please make sure that you already know how to install KubeSphere with a multi-node cluster by following the [guide](https://github.com/kubesphere/kubekey). For the detailed information about the config yaml file that is used for installation, see Multi-node Installation. This tutorial focuses more on how to configure load balancers.
+- You need a VMware vSphere account to create VMs.
+- Considering data persistence, for a production environment, we recommend you to prepare persistent storage and create a StorageClass in advance. For development and testing, you can use the integrated OpenEBS to provision LocalPV as the storage service directly.
+
+## 2. Architecture
+
+![Architecture](/images/docs/vsphere/kubesphereOnVsphere-zh-architecture.png)
+
+## 3. Prepare Linux Hosts
+
+This tutorial  creates 9 virtual machines with **CentOS Linux release 7.6.1810 (Core)**, the default minimal installation, each configuration is 2 Core 4 GB 40 G.
+
+
+| Host IP | Host Name | Role |
+| --- | --- | --- |
+|10.10.71.214|master1|master1, etcd|
+|10.10.71.73|master2|master2, etcd|
+|10.10.71.62|master3|master3, etcd|
+|10.10.71.75|node1|node|
+|10.10.71.76|node2|node|
+|10.10.71.79|node3|node|
+|10.10.71.67|vip|vip|
+|10.10.71.77|lb-0|lb（keepalived + haproxy）|
+|10.10.71.66|lb-1|lb（keepalived + haproxy）|
+
+
+
+![create](/images/docs/vsphere/kubesphereOnVsphere-en-0-1-create.png)
+
```

Comment:
Please add the corresponding description for each step and screenshot. Currently, it looks not that clear in this section.